### ISSUE TITLE
`sched.ch_duration` should report `ch_stop_time - ch_start_time`

### ISSUE THREAD

**Issue by ajavadia** [Qiskit/qiskit-terra#4848]
The duration for a channel is now reported the same as its stop time. I believe this should be stop_time - start_time to actually be the duration.

**Comment by abhik-99** [comment]
Hey @ajavadia, if it's okay, can I help out on this on?

**Comment by Arijit02** [comment]
@ajavadia Can I work on the issue?

**Comment by ajavadia** [comment]
Yes please go ahead, thanks

**Comment by Arijit02** [comment]
I went through the code base and found that there have been certain tests written in test_schedule.py for sched.ch_duration(). I have to modify them also right?

**Comment by ajavadia** [comment]
> I went through the code base and found that there have been certain tests written in test_schedule.py for sched.ch_duration(). I have to modify them also right?

yes the tests will have to change to reflect the new behavior. also a release note is required to document the change.

**Comment by taalexander** [comment]
Just FYI, changing this functionality may break existing code. Tests should
highlight breakages.

**Comment by jakelishman** [comment]
Closing as obsolete
